cg-charter.html

```diff
@@ -224,6 +226,16 @@ <h3 id="test-suites">
       "http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">
       W3C Software and Document License</a>.
     </p>
+    <p>
+      The Community Group will develop a continuous integration mechanism to validate the
+      grammar of WebGPU Shading Language against syntactic ambiguities and potential
+      mechanical issues with primary reliance on an open-source, commercially-permissive
+      license tool that can report ambiguity and generate playgrounds that can run on the Web
+      so that community can leverage and extend easily to enhance the ecosystem. Such software
+      will be developed using the <a href=
+      "http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">
+      W3C Software and Document License</a>.
+    </p>
```

Comment:
I know that this is basically done, but I think this should unfortunately be "may" and not "will", so as not to compel us to work on non-primary deliverables. I'm happy to touch on this on a WGSL call though, and see what we think as a group.